network-config-analyzer/PeerContainer.py

```diff
@@ -279,6 +291,7 @@ def _add_wep_from_yaml(self, wep_object):
         wep_namespace = self.get_namespace(metadata.get('namespace', 'default'))
         wep_name = spec.get('pod', '')
         wep = Pod(wep_name, wep_namespace)
+        # TODO: should handle Pod's serviceAccountName here? how?
```

Comment:
A wep definition includes a `pod` field which points to the corresponding k8s Pod resource.
We can defer handling this for now, maybe just update the TODO.

Reply:
done